quick_start/my_text_classifier_bert.jsonnet

```diff
@@ -0,0 +1,48 @@
+local bert_model = "bert-base-uncased";
+
+{
+    "dataset_reader" : {
+        "type": "classification-tsv",
+        "tokenizer": {
+            "word_splitter": "bert-basic"
+        },
+        "token_indexers": {
+            "bert": {
+                "type": "bert-pretrained",
+                "pretrained_model": bert_model,
+            }
+        }
+    },
+    "train_data_path": "data/movie_review/train.tsv",
+    "validation_data_path": "data/movie_review/dev.tsv",
+    "model": {
+        "type": "simple_classifier",
+        "embedder": {
+            "allow_unmatched_keys": true,
+            "embedder_to_indexer_map": {
+                "bert": ["bert", "bert-offsets"]
+            },
+            "token_embedders": {
+                "bert": {
+                    "type": "bert-pretrained",
```

Comment:
This also should be `pretrained-transformer`.

Reply:
This worked, although it doens't have a paramter `requires_grad` which is necessary for fixing the bert parameters. Do you know if this is being worked on, or am I missing something?

Reply:
Why do you want to fix the bert parameters?  We should add that parameter, but you typically don't want to fix the parameters when using bert.